Dockerfile

```diff
@@ -1,7 +1,33 @@
 FROM python:3.7-slim-buster
 ENV PYTHONUNBUFFERED 1
-WORKDIR /code
-COPY ./requirements/base.txt /code/
-RUN pip install -r base.txt
-COPY celerylogger/celery_logger.py /code/
-CMD python celery_logger.py
+
+ENV CELERY_LOGGER_BROKER=""
+ENV CELERY_LOGGER_LOG_FILE=""
+
+WORKDIR /usr/src/app
+
+# Updating dependencies and installing `gosu`
+RUN set -eux; \
+    apt-get update && \
+    apt-get install -y --no-install-recommends gosu && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+# Copying package files...
+COPY ./requirements/ /usr/src/app/requirements/
+COPY ./setup.py /usr/src/app/
+COPY ./README.md /usr/src/app/
+COPY celerylogger/ /usr/src/app/celerylogger/
+
+# Creating celerylogger user and adjusting permissions
+RUN set -eux; \
+    groupadd --system --gid=888 celerylogger && \
+    useradd --system --gid=celerylogger --uid=888 \
+            --home-dir=/usr/src/app \
+            --shell /bin/bash celerylogger && \
+    chown -R celerylogger:celerylogger /usr/src/app
+
+# Installing dependencies using the celerylogger user
+RUN pip install .
+
+# Running with the limited user permissions
+CMD gosu celerylogger celery-logger
```

Comment:
It's usually a better idea to use list instead of pure string. Pure string spawns a new shell (shell form) while the list will use the PID 1 (vector form).

Reply:
Thanks!